op-guide/history-read.md

```diff
@@ -20,7 +20,7 @@ The `tidb_snapshot` system variable is introduced to support reading history dat
 - The variable is valid in the `Session` scope.
 - Its value can be modified using the `Set` statement. 
 - The data type for the variable is text. 
-- The variable is to record time in the following format: “2016-10-08 16:45:26.999”. Generally, the time can be set to seconds like in “2016-10-08 16:45:26”. 
+- The variable can store TSO (Timestamp Oracle) and time. TSO is a globally unique time service, which obtains information from PD. The time that this variable records uses the following format: "2016-10-08 16:45:26.999". Generally, the time can be set to just seconds like in "2016-10-08 16:45:26".
```

Comment:
The variable accepts TSO (Timestamp Oracle) and datetime. TSO is a globally unique time service, which is obtained from PD. The acceptable datetime format is "2016-10-08 16:45:26.999". Generally, the datetime can be set using second precision, e.g, "2016-10-08 16:45:26".